content/pt-br/docs/reference/glossary/istio.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Istio
+id: istio
+date: 2018-04-12
+full_link: https://istio.io/docs/concepts/what-is-istio/
+short_description: >
+  Uma plataforma aberta (não específica do Kubernetes) que fornece uma maneira uniforme de integrar microsserviços, gerenciar o fluxo de tráfego, aplicar políticas e agregar dados de telemetria.
+
+aka: 
+tags:
+- networking
+- architecture
+- extension
+---
+  Uma plataforma aberta (não específica do Kubernetes) que fornece uma maneira uniforme de integrar microsserviços, gerenciar o fluxo de tráfego, aplicar políticas e agregar dados de telemetria.
+
+<!--more--> 
+
+Adicionar o Istio não requer alterar o código da aplicação. É uma camada de infraestrutura entre um serviço e a rede, quando combinada com os serviços de implantações, é comumente referida como uma malha de serviço. A camada de gerenciamento do Istio abstrai a plataforma de gerenciamento do cluster principal, que pode ser Kubernetes, Mesosphere, etc.
```

Comment:
"serviços de implantações" in this sentence is a bit far from the original sentence. Perhaps "instalações de serviços" would be a better fit here?